src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/Globals.cs

```diff
@@ -778,35 +778,20 @@ internal static Type TypeOfXmlNodeArray
             }
         }
 
-        private static bool s_shouldGetDBNullType = true;
-
-        private static Type s_typeOfDBNull;
+        private static Type s_typeOfDBNull = typeof(DBNull);
```

Comment:
We could probably update the sites directly.  I will see how many places that is.